### PR TITLE
Feat: Add font sizes to the Styleguide

### PR DESCRIPTION
**Pull request by jimhunty**
Originally Jake did an audit on the site to check font sizes. As a result he has decided to change some of these. We are also ignoring the Android hacks in favour of consistent font sizes as per the change in strategy last year. As a result some snapshots needed updating to reflect this.

You will notice duplicate names, this is in order to give the correct components the correct name that could potentially change in future. Therefore making this more maintainable. The list also corresponds closer to the elements that Jake designs for.

### CODE REVIEW

File: packages/styleguide/fonts/font-sizes.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+const fontSizes = {
```
[the remark this answers is not in the document]
Comment (jimhunty) [reply packages/styleguide/fonts/font-sizes.js]: The latter seems to long to me, but the grouping of headline works well. How would you consider `body`?

Anyone else have any preference?

[the remark this answers is not in the document]
Comment (craigbilner) [reply packages/styleguide/fonts/font-sizes.js]: alphabetical!

[the remark this answers is not in the document]
Comment (jimhunty) [reply packages/styleguide/fonts/font-sizes.js]: @craigbilner Surely size order works best here?

[the remark this answers is not in the document]
Comment (aronshamash) [reply packages/styleguide/fonts/font-sizes.js]: @jimhunty `font.size.headline.article` is same length as `fontSizes.headline.article` and means we could have `font.spacing` ?

I prefer size order too.

[the remark this answers is not in the document]
Comment (jimhunty) [reply packages/styleguide/fonts/font-sizes.js]: Surely spacing is more generic and not just for fonts? And I meant deeper nesting and more potential branches, which increases complexity.

[the remark this answers is not in the document]
Comment (aronshamash) [reply packages/styleguide/fonts/font-sizes.js]: yes sorry spacing example is wrong.

[the remark this answers is not in the document]
Comment (craigbilner) [reply packages/styleguide/fonts/font-sizes.js]: @jimhunty it's hard to find what you're after and hard for devs to know immediately where to put a property, alphabetical is easy to read, easy to follow and non-controversial

[the remark this answers is not in the document]
Comment (jimhunty) [reply packages/styleguide/fonts/font-sizes.js]: Changed

**Pull request merged**